django-s3-file-field-client/src/client.ts

```diff
@@ -49,36 +49,31 @@ export default class S3FFClient {
    * @param chunk The data to upload to this part.
    * @param part Details of this part.
    */
-  protected async uploadPart(chunk: ArrayBuffer, part: PartInfo): Promise<UploadedPart> {
+  protected async uploadPart(chunk: Blob, part: PartInfo): Promise<UploadedPart> {
     const response = await axios.put(part.upload_url, chunk);
-    const etag = response.headers['etag'];
+    const etag = response.headers.etag;
     return {
       part_number: part.part_number,
       size: part.size,
       etag,
-    }
+    };
   }
 
   /**
-   * Uploads all the parts in a file directly to an object store in parallel.
+   * Uploads all the parts in a file directly to an object store in serial.
    *
    * @param file The file to upload.
    * @param parts The list of parts describing how to break up the file.
    */
   protected async uploadParts(file: File, parts: PartInfo[]): Promise<UploadedPart[]> {
-    const buffer = await file.arrayBuffer();
```

Comment:
@brianhelba please note I removed the conversion to `ArrayBuffer` because file supports `slice` itself. If there was some intentional reason to use ArrayBuffer instead, please let me know.

Reply:
Ping @dchiquit , who wrote this.

Reply:
I was not aware file could slice.

Reply:
🔪